xs2a-adapter-aspsp-registry/src/main/java/de/adorsys/xs2a/adapter/registry/AspspCsvServiceImpl.java

```diff
@@ -57,7 +57,7 @@ private String toCsvString(AspspCsvRecord aspsp) {
         try {
             return mapper.writer(schema).writeValueAsString(aspsp);
         } catch (JsonProcessingException e) {
-            LOG.warn("Exception occurred while indexes were being written into a CSV" + e.getMessage());
+            LOG.warn("Exception occurred while indexes were being written into a CSV: " + e.getMessage());
```

Comment:
1) rename `LOG` into `log`
2) try to use place holders instead of a concatenation of the string
`log.warn("Exception occurred while indexes were being written into a CSV: {}", e.getMessage());`